src/exceptions.ts

```diff
@@ -10,11 +10,11 @@
 import { createError } from '@poppinss/utils'
 
 export const E_INSECURE_APP_KEY = createError(
-  'The value of "app.appKey" should be atleast 16 charcaters long',
+  'The value of your key should be at least 16 characters long',
```

Comment:
if we change the content of these messages ( and i think we should ), maybe we should also add a check in the core, and throw a more explicit error message in case appKey is invalid.

See : https://github.com/adonisjs/core/blob/feat/configure-install-pkg/providers/app_provider.ts#L94

because I think a lot of people won't understand if we don't specify explicitly that it comes from config/app.appKey

Reply:
It will not come from `app.appKey` anymore. The key will be defined directly inside the `encryption` config file.

You will be able to define a key per driver/encrypter so you can easily rotate your key if needed.

Reply:
Oooh yeah okay, makes sense 👌